app/controllers/guide/scenarios_controller.rb

```
@@ -2,8 +2,12 @@ class Guide::ScenariosController < Guide::BaseController
   layout 'guide/scenario'
 
   def show
-    expose_layout
-    expose_scenario
+    begin
+      expose_layout
+      expose_scenario
+    rescue NoMethodError => e
+      render text: 'Nothing to see here.', status: '404'
```

It may be worth logging `e` rather than silently dropping it.